ueransim/etc/ueransim/open5gs-ue.yaml

```diff
@@ -0,0 +1,55 @@
+snn: '5G:mnc001.mcc01.3gppnetwork.org'
```

Comment:
remove open5gs name from open5gs-gnb.yaml configuration. We should not make references to a specific core in the ueransim image